loader_hub/reddit/requirements.txt

```diff
@@ -1,9 +1,5 @@
-certifi==2022.12.7
```

Comment:
@oraluben why remove these requirements?

Reply:
This will be installed via requests and I see no need to keep it.

Reply:
ah ok, makes sense